frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentHeader/FeatureOverviewEnvironmentToggle/FeatureOverviewEnvironmentToggle.tsx

```diff
@@ -1,17 +1,25 @@
+import { styled } from '@mui/material';
 import { useFeatureToggleSwitch } from 'component/project/Project/ProjectFeatureToggles/FeatureToggleSwitch/useFeatureToggleSwitch';
 import { FeatureToggleSwitch } from 'component/project/Project/ProjectFeatureToggles/FeatureToggleSwitch/FeatureToggleSwitch';
 import { useChangeRequestsEnabled } from 'hooks/useChangeRequestsEnabled';
 import { useFeature } from 'hooks/api/getters/useFeature/useFeature';
 import { useRequiredPathParam } from 'hooks/useRequiredPathParam';
 import type { IFeatureEnvironment } from 'interfaces/featureToggle';
 
-interface IFeatureOverviewEnvironmentToggleProps {
-    environment: IFeatureEnvironment;
-}
+const StyledContainer = styled('div')(() => ({
+    order: -1,
```

Comment:
Ooooh, I wanna be real careful about this. After my deep dive on ordering the other day, I'm not sure this is what we wanna do. How does this affect tab order?

From [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/order):

> Using the order property will create a disconnect between the visual presentation of content and DOM order. This will adversely affect low vision users navigating with the aid of assistive technology such as a screen reader. If the visual order differs from the DOM order, your users will have different experiences depending on how they access your content.

Can we do this without order?

Reply:
Yup. We do not have navigation elements in between, and I think it is an issue for screen readers, to read environment name first and toggle later.

Reply:
Hmm, might be. But if this is on the feature page, why would they even need the flag name? 🤔 But yeah, I think I support your motion here.